### ISSUE TITLE
SonarView - Option to remove Grid Circles

### ISSUE THREAD

**Issue by KevinKW87** [CeruleanSonar/SonarView#4]
Request a new Display Control option to remove circular grid lines from sonar image for cleaner sonar image when screenshotting.

**Comment by ljlukis** [comment]
Good idea!Sent from Larry's iPhoneOn Aug 27, 2024, at 10:02 PM, Kevin Klemens ***@***.***> wrote:﻿
Request a new Display Control option to remove circular grid lines from sonar image for cleaner sonar image when screenshotting.
